app/assets/javascripts/classes/KnockoutBindings.ts

```diff
@@ -47,6 +50,53 @@ module kobindings {
         }
     };
 
+    ko.bindingHandlers['popover'] = {
```

Comment:
So bootstrap is available here for whatever reason but I suspect you're right in that it's probably not being pulled in as a direct dependency here.

I've added an import and I now see it in the generated javascript but I think I'll make an issue to look over its usage since it's possibly being loaded in everywhere.

See https://stackoverflow.com/questions/22762890/typescript-9-5-import-bootstrap-into-an-amd-module-to-use-jquery-extensions